docs/development.adoc

```diff
@@ -158,34 +158,25 @@ $ MINISHIFT_ENABLE_EXPERIMENTAL=y minishift start --extra-clusterup-flags "--ena
 
 * `odo` and `oc` binaries in `$PATH`.
 
-.Procedure:
-
-To deploy an integration test:
+.How to write:
 
+Refer odo clean test link:https://github.com/openshift/odo/blob/master/tests/template/template_cleantest_test.go[`template`].
 
-* For the entire integration suite use:
-----
-$ make test-e2e
-----
+.Procedure:
 
-* For the generic tests use:
-----
-$ make test-generic
-----
+Integration tests can be run in the following two ways:
 
-* For the component tests use:
+* To run component test spec in parallel (default) on `4` test node :
```

Comment:
Sentence is a bit confusing.

Maybe you mean `To run the component test in parallel (default: 4), on a test cluster:` ?

Reply:
https://github.com/openshift/odo/pull/1759#discussion_r290604136